packages/react/src/components/CollapsibleNavbarItem/CollapsibleNavbarItem.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). 
```

Reply:
This needs to be fixed in a different repo.